examples/tests/microbit_v2/Makefile

```diff
@@ -0,0 +1,16 @@
+# Makefile for user application - copied from bradjc's Hail test app
+
+# Specify this directory relative to the current application
+TOCK_USERLAND_BASE_DIR = ../../..
+
+# Which files to compile
+C_SRCS := $(wildcard *.c)
+
+#APP_HEAP_SIZE := 2048
+
+# External libraries used
+EXTERN_LIBS += $(TOCK_USERLAND_BASE_DIR)/simple-ble
```

Comment:
```suggestion

```

Reply:
I don't believe we're using `simple-ble` anymore, are we?

Reply:
Or maybe we are for "gap.h"?

Reply:
Yep. I just double-checked, and the simple-ble lines are needed for gap.h